charm_settings.py

```diff
@@ -0,0 +1,17 @@
+from netbox.original_settings import *
```

Comment:
It is generally not recommended to import `*`, perhaps this is required by netbox?

Reply:
This situation will happen in most 12 factor Django autogenerated projects, so it maybe an interesting point to discuss

Reply:
I understand why it is done here so for this PR we can treat this as resolved. I'm curios why this would also be needed in other 12-factor projects? It seems like netbox publishes a `settings.py` file and we would like to modify it without impacting the upstream, in most cases it would be fine to edit the default `settings.py` since the app is owned by the project?

Reply:
I think the case we're looking at here could be more common than you think (I hope so), where 12 factor becomes a way for a pre-existing upstream project to be deployed rather than something that was explicitly written to work with the 12 factor project (and so `settings.py` can just be changed directly).